### PR TITLE
Forward --force-overwrite option to underlying `neuro-extras image build`

### LINKED ISSUE #203: Support docker image overwrite
By default in neuro-extras (starting from v20.11.13), we prohibit image build if the destination image already exists.
To enforce overwrite one should pass `-F, --force-overwrite` flag to `neuro-extras image build` command.

Let's think of supporting this flag in neuro-flow.
We could support it on the recipes (YAML files) level or in CLI. Or even both.
For me, that CLI-only seems to be the best option since it is rather an exception than a regular workflow.

@asvetlov, what do you think?

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**
Resolves #203

### CODE REVIEW

File: neuro_flow/cli/images.py [hunk 1]
```diff
@@ -16,9 +16,16 @@
 
 
 @click.command()
+@option(
+    "-F",
```
Comment (asvetlov) [inline neuro_flow/cli/images.py:20]: Usually this flag is called `-f/--force`. Do you have a reason for different naming?

**Review by asvetlov — COMMENTED** [review]
The diff for `reference/cli.md` is too huge.
Perhaph you need sync with master and run `make docs`.

File: neuro_flow/cli/images.py (hunk 1 above)
Comment (YevheniiSemendiak) [reply neuro_flow/cli/images.py:20]: The only reason is to sync the flag naming with [neuro-extras](https://github.com/neuro-inc/neuro-extras/blob/master/neuro_extras/main.py#L104), but we could naturally call it `-f/--force` in neuro-flow.

**Comment by YevheniiSemendiak** [comment]
`make docs` is run automatically due to pre-commit-hook before each commit.
I've tried to run it once again, but didn't get any difference:
image

Maybe, I'm doing something wrong?

**Comment by romasku** [comment]
I think it's because `gitbook` took control over `cli.md` file and it probably reformated it. I think we can ignore those changes.

**Comment by asvetlov** [comment]
Ok, let's see how it works

**Review by asvetlov — APPROVED** [review]

**Pull request merged**